types/react-reconciler/test/ReactTestHostConfig.ts

```diff
@@ -1,189 +1,204 @@
-import * as TestRendererScheduling from './ReactTestRendererScheduling';
+/**
```

Comment:
I'm somewhat confused by this header.  Is the test copyright Facebook?  Is there a license file?  I'm pretty sure this isn't flow.

Reply:
Yes, it's license header, `@flow` is removed